docs/DeveloperGuide.md

```diff
@@ -60,16 +64,37 @@ The figure below represents the sequence diagram when the user wants to add a st
 
 <>
 
+### Ui
+
+The ui component only contains the TextUi.java file and its API can be found
+[here](https://github.com/AY2122S1-CS2113T-T12-1/tp/blob/master/src/main/java/seedu/mtracker/ui/TextUi.java).
+
+It is a basic java class containing private string attributes of frequently used display texts.
```

Comment:
Just say its a basic java class that contains the helper methods for displaying the different features to the user
^something like this. Reason is because java class containing private string attributes does not seem completely accurate, as we have generally have helper methods of that sort